prep-css-flexbox/flex-wrap.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+  [id~="card-deck"] {
+    display: flex;
+    flex-wrap: wrap;
+    flex-basis: calc(100% / 6);
+    padding: 5px;
+  }
+
+  [class~="card-container"] {
```

Comment:
Ok, very good. There is actually some dedicated syntax for selecting elements by `id` attribute or `class` attribute.

To select an element by id:

```css
```